eng/testsos.cmd

```diff
@@ -0,0 +1 @@
+%~dp0..\.dotnet\dotnet.exe test --no-build --logger "console;verbosity=detailed" %~dp0..\src\SOS\SOS.UnitTests\SOS.UnitTests.csproj
```

Comment:
Does this work? Usually, you need to be in the folder that has the config.

Reply:
Yes, it seems to work just fine.